projects/dsp-ui/src/lib/viewer/values/text-value/text-value-as-xml/text-value-as-xml.component.scss

```diff
@@ -6,6 +6,10 @@
   font-family: monospace;
```

Comment:
@kilchenmann I have just notice that these styles are overridden by the app's main.scss:

```css
* {
  font-family: Arial, Helvetica, sans-serif;
}

```

 Any idea what could be done about that?

Reply:
Hm, we shouldn't use `* {` in this css. But it can happen in all apps.

Reply:
`ng-deep` is anyway deprecated since Angular v6. Could you add the two setups in ng-deep to the viewer.scss?

```scss
.ck-content code {
  font-family: monospace;
}
.ck-content pre {
   background-color: inherit;
 }
```

But not sure if this helps...

Reply:
@tobiasschweizer I just updated the code in the previous comment